the-collab-lab/tcl-68-smart-shopping-list
# Sm dg style header navigation

## Description

This PR adds styling to our header and navbar at the bottom of the app, according to our prototype / mockup styling. 

## Related Issue

Closes Issue #57

## Acceptance Criteria

- [x] Logo/Header
- [x] `Navigation Bar`
- [x] `Buttons`: Navigation Bar
- [x] (After Sign In) `Header text`: Signed in As
- [x] (After Sign In) `Button`: Sign Out

## Type of Changes

`enhancement`

## Updates

### Before

![Screenshot 2024-04-03 at 8 04 22 PM](https://github.com/the-collab-lab/tcl-68-smart-shopping-list/assets/117242156/e8fd773a-9571-4f69-847b-d0cfd866fc52)


### After

<img width="372" alt="image" src="https://github.com/the-collab-lab/tcl-68-smart-shopping-list/assets/137969744/70a46858-ad25-4a17-8c29-55bc64f3afdc">



## Testing Steps / QA Criteria

1. Navigate to deployment below
2. Confirm that header, logo and nav buttons are in same style as 

Note: There is spacing added to the 'Signed in as' and button next to it on our other PR (home page styling) as well as the background color changed to eggshell on the body as well (those changes not represented in these screenshots).
